src/tests/functional/page_refresh_stream_action_tests.js

```diff
@@ -9,24 +9,24 @@ test.beforeEach(async ({ page }) => {
 test("test refreshing the page", async ({ page }) => {
   assert.match(await textContent(page), /Hello/)
 
-  await page.locator("#content").evaluate((content)=>content.innerHTML = "")
+  await page.locator("#content").evaluate((content) => content.innerHTML = "")
   assert.notMatch(await textContent(page), /Hello/)
 
   await page.click("#refresh button")
-  await nextBeat()
+  await sleep(250)
```

Comment:
Since we'll add an option, you could replace 250 with `Turbo.session.pageRefreshDebouncePeriod`. Also, since we do it in several tests, I'd consider extracting a local test helper function `await nextPageRefresh()` that encapsulates the `sleep` and clarifies why that active wait is due.

Reply:
Sure, I've extracted it, what do you think now?